pkg/hintrunner/hint.go

```diff
@@ -215,3 +215,47 @@ func (hint WideMul128) Execute(vm *VM.VirtualMachine) error {
 
 	return nil
 }
+
+type DebugPrint struct {
+	start ResOperander
+	end   ResOperander
+}
+
+func (hint DebugPrint) Execute(vm *VM.VirtualMachine) error {
+	var startAddr memory.MemoryAddress
+	switch v := hint.start.(type) {
+	case Deref:
+		startAddr, _ = v.deref.Get(vm)
+	default:
+		return fmt.Errorf("start has to be a pointer")
+	}
+
+	var endAddr memory.MemoryAddress
+	switch v := hint.end.(type) {
+	case Deref:
+		endAddr, _ = v.deref.Get(vm)
+	default:
+		return fmt.Errorf("end has to be a pointer")
+	}
+
+	if startAddr.Offset > endAddr.Offset {
+		return fmt.Errorf("start cannot be greater than end")
+	}
+
+	current := startAddr.Offset
+
+	for current < endAddr.Offset {
+		v, err := vm.Memory.ReadFromAddress(&memory.MemoryAddress{
+			SegmentIndex: startAddr.SegmentIndex,
+			Offset:       current,
+		})
```

Comment:
After reading some, ouptut segment only contains felts, so no need to add hex method to anyone. You can assume your extracting a field element

Reply:
Okay, removed both and used `felt.Text(16)` to it in hex format